sklearn/tests/test_pipeline.py

```diff
@@ -1813,6 +1815,102 @@ def test_pipeline_inverse_transform_Xt_deprecation():
         pipe.inverse_transform(Xt=X)
 
 
+# transform_input tests
```

Comment:
I would like to see one very simple test where the exact expected output of the new transformed metadata is tested for.
`test_transform_input_pipeline` is already quite indirect (but a good test).